eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -141,6 +143,18 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->registerForAutoconfiguration(QueryType::class)
             ->addTag(QueryTypePass::QUERY_TYPE_SERVICE_TAG);
+
+        $container->registerForAutoconfiguration(ConfigScopeChangeSubscriber::class)
+                  ->addTag(
+                      'kernel.event_listener',
+                      ['method' => 'onConfigScopeChange', 'event' => MVCEvents::CONFIG_SCOPE_CHANGE]
+                  );
+
+        $container->registerForAutoconfiguration(ConfigScopeChangeSubscriber::class)
+                  ->addTag(
+                      'kernel.event_listener',
+                      ['method' => 'onConfigScopeChange', 'event' => MVCEvents::CONFIG_SCOPE_RESTORE]
+                  );
```

Comment:
```suggestion
        $container->registerForAutoconfiguration(ConfigScopeChangeSubscriber::class)
              ->addTag(
                  'kernel.event_listener',
                  ['method' => 'onConfigScopeChange', 'event' => MVCEvents::CONFIG_SCOPE_CHANGE]
              )
              ->addTag(
                  'kernel.event_listener',
                  ['method' => 'onConfigScopeChange', 'event' => MVCEvents::CONFIG_SCOPE_RESTORE]
              );
```

Indentation and AFAIK can be chained.